packages/learner-access/src/components/LoadUserLearning.tsx

```diff
@@ -237,15 +236,15 @@ const LoadUserLearning = ({ query, kind, sort }: LoadedComponentProps): JSX.Elem
                 }}
                 className="bg-active-blue text-white rounded-sm cursor-pointer inline-block font-normal text-xs m-0 py-[0.15rem] px-4 relative text-center no-underline ease-in-out border-active-blue font-sans transition duration-200 leading-5"
               >
-                {item.availabilityStatus == 'completed'
+                {item.callToAction == 'completed'
```

Comment:
@Jamal-majid-ti Apologies for the comment clarity; I meant the entire block could be replaced with `{item.callToAction}`. There doesn't need to be any status checks, as `getCallToAction` will handle status checks, content types, etc., and return the appropriate translation:

https://github.com/thoughtindustries/helium/blob/2f6f495301bd7c4b596a19b1d5a5112a97ce4a56/packages/content/src/utilities/hydrate-content/hydrate-content.ts#L154